.vscode/launch.json

```diff
@@ -9,9 +9,9 @@
             "request": "launch",
             "name": "Launch Program",
             "program": "${workspaceFolder}/index.js",
-            "args": ["-d","-t","@newsIsNotVisible" ],
+            "args": ["-d","-t","@adminLogin" ],
             "console":"integratedTerminal",
-            "runtimeVersion": "10.15.3",
+          //  "runtimeVersion": "10.15.3",
```

Comment:
I'm not sure why the vscode project is checked in, but I guess you should revert your local changes before merging to keep it clean and not disturb other devs

Reply:
please also add it to .gitignore

Reply:
thanks for the finding, was rather unwanted, I reverted my changes